keyboards/draytronics/daisy_v2/keyboard.json

```diff
@@ -0,0 +1,86 @@
+{
+    "manufacturer": "Draytronics",
+    "keyboard_name": "DAISY",
+    "maintainer": "ghostseven",
+    "bootloader": "stm32-dfu",
+    "bootmagic": {
+        "enabled": true
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "enabled": true,
+        "rotary": [
+            {"pin_a": "A15", "pin_b": "A14"}
+        ]
+    },
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": true,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false,
+        "oled": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["B11", "B10", "B2", "B1"],
+        "rows": ["A2", "A1", "A0"]
+    },
+    "mouse_key": {
+        "enabled": true
+    },
+    "processor": "STM32F072",
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        },
```

Comment:
Given that the default keymap does not contain any modifier keys, I don't think this is needed.
```suggestion
```